src/util.ts

```diff
@@ -129,3 +131,24 @@ export async function download(url: string): Promise<string> {
     req.end();
   });
 }
+
+async function getFiles(filePath :string) {
+  const entries = await promises.readdir(filePath, { withFileTypes: true });
+
+  // Get files within the current directory
+  const files = entries
+    .filter(file => (!file.isDirectory() && file.name.endsWith('.k8s.yaml')))
+    .map(file => (filePath + '/' + file.name));
+
+  // Get sub-folders within the current folder
+  const folders = entries.filter(folder => folder.isDirectory());
+
+  for (const folder of folders) {
+    /*
+        Get files in sub-folders
+      */
```

Comment:
We can remove this comment

Reply:
I made the changes to remove the comment and annotate the function with the return type.
On adding the unit tests, sure thing. However I was looking for the tests for the existing functionality as I could probably reuse most of that but I cannot find it. Could you point me to it? I see a test/synth folder but that does not seem to be it
Thanks

Reply:
I'm not sure if there are any existing tests for this. Either way `getFiles` is self-contained so you can just put any tests in a new test file (like `util.test.ts`) within the tests directory.

It should be possible to use code like this to create temporary directories and files within a test case:

```ts
const basedir: string = await fs.mkdtemp(path.join(os.tmpdir(), 'something'));
const subdir: string = await fs.mkdirp(path.join(basedir, 'something-else'));
await fs.writeFile(path.join(basedir, 'example.yml'), 'contents');
```

Reply:
@Chriscbr I added the test for the new function

Thanks!